app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -41,7 +48,7 @@ data class Event(
       mainOrganiser = map["main_organiser"] as String,
       organiserList = convertToMutableListOfStrings(map["organisers_list"]),
       attendeeList = convertToMutableListOfStrings(map["attendees_list"]),
-      category = EventCategory.valueOf(map["category"] as String),
+      category = EventCategory.fromDisplayName(map["category"] as String) as EventCategory,
```

Comment:
this will also crash the app. the actual event category value (e.g. "SOCIAL", "COMMUNITY") is stored, not the display name.

Reply:
Will remove the fromDisplayName function, I didn't realise that valueOf already did the same, oops!